apps/dashboard/app/apps/user_preferences.rb

```diff
@@ -0,0 +1,43 @@
+class UserPreferences
+    require 'yaml'
+  
+    def initialize()
+      _prefs
+    end
+  
+    def preferences_file
+        @preferences_file ||= begin
+            file = ENV['OOD_PREFERENCES_FILE'] || "#{User.new.dir}/preferences.yml"
+            Pathname.new(file.to_s).expand_path
+        end
+    end
+    
+    private def _prefs
+        @_prefs ||= begin
```

Comment:
I know you took this from me, but I don't think I'm in love with `_pref`.  Let's just have an attr_reader for `preferences` then work on it. `UserPreferences.preferences` is a little wonky, but it'll work for now until we think of something better.